cvelib/cli.py

```diff
@@ -176,6 +176,7 @@ def wrapped(*args: Any, **kwargs: Any) -> Callable:
             click.echo("CVE record is not valid against the v5 JSON schema:")
             for error in exc.errors:
                 click.echo(f"  {error}")
+        click.echo("\nUse `cve <command> -h` for more information on how to use this command")
```

Comment:
LGTM as is!

If you wanted to make it spit out the exact command being called, you could use the following suggestion :wink:
```suggestion
        cmd_ctx = next((arg for arg in args if isinstance(arg, click.Context)), None)
        cmd_name = cmd_ctx.command.name if cmd_ctx else "<command>"
        click.echo(f"\nUse `cve {cmd_name} -h` for more information on how to use this command")
```
Ex:
```
> cve reject CVE-2024-20699
ERROR: 403 Client Error: Forbidden for url: https://cveawg-test.mitre.org/api/cve-id/CVE-2024-20699?state=REJECTED
DETAILS: {'error': 'CANNOT_CHANGE_CVE_ID_WITH_RECORD', 'message': 'A record was found for CVE-2024-20699. A CVE ID cannot be changed once a record exists for it. Instead, it changes according to the record.'}

Use `cve reject -h` for more information on how to use this command
```